ote/src/cljs/ote/views/service_search.cljs

```diff
@@ -219,10 +219,11 @@
              [:span (tr [:service-search :participating-operator])]]))])]))
 
 (defn results-listing [e! {service-search :service-search user :user :as app}]
+  (let [{:keys [results empty-filters? total-service-count filter-service-count fetching-more?]} service-search
```

Comment:
Looks like won't evaluate properly, code integration issue?

Reply:
I forgot to push one commint :)